packages/contracts-ts/scripts/version.js

```
@@ -0,0 +1,9 @@
+const fs = require('fs')
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)